### ISSUE TITLE
Improve `no_empty_blocks`

### ISSUE THREAD

**Issue by arthurbcd** [solid-software/solid_lints#172]
Some empty blocks such as `setState` should and can be empty in some cases:
https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#comment-empty-closures-to-setstate

I suggest adding an option to resolve the lint when adding any comments, other than just TODO.

```yaml
no_empty_blocks:
  allow_todos: true
  allow_comments: true
```

It would be also interesting to improve the linter message, instead of just "Block is empty...".

Depending on the configuration, we could say something like:
LINTER: "Empty blocks are often indicators of missing code. Either implement it or add a TODO/comment."

I can see this as a really strong lint to find all empty blocks when needed, by just changing those values of the linter.

Thanks for this linter. Has been really useful for me and my team. :)
